antora-ui-camel/src/css/frontpage.css

```diff
@@ -270,6 +269,17 @@ section.frontpage h2 {
     height: 10rem;
   }
 
+  .split {
+    padding: 0 2rem;
```

Comment:
```suggestion
    margin: 0 1rem;
```

Reply:
yes, I think the margin looks a bit much for the mobile screen. Made the suggested changes.